multibranch/src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java

```diff
@@ -123,6 +127,7 @@
         // RateLimitBranchProperty & BuildRetentionBranchProperty hidden by JobPropertyStep.HideSuperfluousBranchProperties.
         // UntrustedBranchProperty hidden because it applies only to Project.
         assertEquals(Collections.<Class<? extends BranchProperty>>emptySet(), clazzes);
+        assertEquals(Collections.<BranchPropertyStrategyDescriptor>emptyList(), r.jenkins.getDescriptorByType(BranchSource.DescriptorImpl.class).propertyStrategyDescriptors(p, r.jenkins.getDescriptorByType(SingleSCMSource.DescriptorImpl.class)));
```

Comment:
I'm ready to see this line without scrolling.

![image](https://cloud.githubusercontent.com/assets/1017585/13794442/144d005c-eafc-11e5-8833-3c3e20e4531a.png)